modules/terrain/docs/api-reference/terrain-loader.md

```diff
@@ -0,0 +1,73 @@
+# TerrainLoader
+
+The `TerrainLoader` parses the OBJ half of the classic Wavefront OBJ/MTL format.
+
+| Loader                | Characteristic                                |
+| --------------------- | --------------------------------------------- |
+| File Extension        | `.png`, `.pngraw`                             |
+| File Type             | Binary                                        |
+| File Format           | Encoded height map                            |
+| Data Format           | [Mesh](/docs/specifications/category-mesh.md) |
```

Comment:
Can I add this to the category-mesh docs?

Reply:
Yes please

Reply:
Yes. Also the front page of the docs has a list of loaders in each category.